plugins/woocommerce/changelog/51286-51176-fixed-fatal-is_visible-func

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+Comment: Fixed call to a member function is_visible() on string | content-product.php:23
```

Comment:
Hi! Why would we not surface this in the changelog?

(Making it a comment means essentially means it will be hidden from the changelog.)

Reply:
Hi @barryhughes ,

Can we use the give on as comment in changelog.

```
Fixed an error in `content-product.php` where a call to `is_visible()` was made on a string instead of a product object, preventing proper visibility checks.
```